listenbrainz/webserver/static/js/src/player-pages/PlayerPage.tsx

```diff
@@ -197,6 +225,18 @@ export default class PlayerPage extends React.Component<
                       />
                     </li>
                   )}
+                  {currentUser?.name && (
```

Comment:
Is there a specific reason for needing the name?
If it's just to make sure we have a user object, let's change it to `auth_token` which would make a bit more sense since it's used in savePlaylist.

Reply:
Done